src/System.IO.Pipelines/src/System.IO.Pipelines.csproj

```diff
@@ -38,11 +36,11 @@
   <ItemGroup Condition="'$(TargetGroup)'=='netcoreapp'">
     <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetCoreApp21.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.1'">
-    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetStandard.cs" />
+  <ItemGroup Condition="'$(TargetGroup)'=='netstandard1.3'">
+    <Compile Include="System\IO\Pipelines\ThreadPoolScheduler.NetStandard13.cs" />
```

Comment:
Nit: This should be lower-case `netstandard1.3.cs` per https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/project-guidelines.md#code-file-naming-conventions

Reply:
Yes please make these lower-case.